components/datetime/src/error.rs

```diff
@@ -30,6 +30,12 @@ pub enum DateTimeFormatterLoadError {
     Data(DataError),
 }
 
+impl From<DataError> for DateTimeFormatterLoadError {
```

Comment:
I'm okay with this, but @robertbastian pushed back on adding the `From` impls for the fine-grained error types in this crate.

Reply:
hm. we kind of need it to use the closure but we could also not use the closure and revive the macro instead for these few call sites.

Reply:
hm. we kind of need it to use the closure but we could also not use the closure and revive the macro instead for these few call sites.